src/mongocxx/search_index_view.hpp

```diff
@@ -86,6 +86,27 @@ class MONGOCXX_API search_index_view {
 
     ///
     /// @{
+    ///
+    /// This is a convenience method for creating a single search index.
+    ///
+    /// @param definition
+    ///    The document describing the search index to be created.
+    ///
+    /// @return The name of the created search index.
+    ///
+    std::string create_one(bsoncxx::document::view_or_value definition);
+
+    ///
+    /// This is a convenience method for creating a single search index.
+    ///
+    /// @param definition
+    ///    The document describing the search index to be created.
+    ///
+    /// @return The name of the created search index.
+    ///
+    std::string create_one(const client_session& session,
+                           bsoncxx::document::view_or_value definition);
+
```

Comment:
These two overloads were added due to changes made in [DRIVERS-2630](https://github.com/mongodb/specifications/commit/267a54d1bb9af894d05d3dd1a6bda0fa53e20a3f#diff-8012d6483484505d56bd53efe58a4f4affbc1512d15d09fcd003241c6ae2fcbeR1049). This change allows for `create_one` to be called while only passing through a definition, no name.